packages/gatsby-source-wordpress/src/steps/source-nodes/create-nodes/create-remote-media-item-node.js

```diff
@@ -138,6 +138,9 @@ export const errorPanicker = ({
       )
     )
     reporter.panic(error)
+  } else {
+    console.error(error)
+    reporter.panic()
```

Comment:
What's wrong with panicking with an error?
```suggestion
    reporter.panic(error)
```

Reply:
reporter.panic() is broken right now - this will result in Gatsby logging "There was an error" and quitting with no additional info https://www.gatsbyjs.com/docs/reference/config-files/node-api-helpers/#reporter